app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,60 @@
+class OrdersController < ApplicationController
+  include OrdersHelper
+
+  before_action :authenticate_user!, only: :add_to_cart
+
+  def authenticate_user!
```

Comment:
This would belong better in application controller as it might be used in multiple places